composer.json

```
@@ -19,22 +19,22 @@
         "neos/neos": "~7.0.0",
         "neos/site-kickstarter": "~7.0.0",
 
-        "neos/demo": "@dev",
-        "neos/neos-ui": "@dev",
+        "neos/demo": "~6.3.0 || 7.0.x-dev",
```

Or should this be without dev?
```suggestion
        "neos/demo": "~6.3.0",
```

---

I would expect "neos/demo": "~7.0.0"

---

But it's not following the same versioning… there is no 7.0.0. 🤷‍♂️